lib/tty/prompt/multi_list.rb

```diff
@@ -56,6 +55,27 @@ def confirm
         super if valid
       end
 
+      def confirm_keys(value = (not_set = true))
+        super
+        check_conflicting_keys
+        @confirm_keys
+      end
+
+      # Set select keys
+      #
+      # @param [Array<Symbol, String, Hash{Symbol, String => String}>] value
+      #   the new select_keys
+      #
+      # @return [Hash{Symbol, String => String}]
+      #
+      # @api public
+      def select_keys(value = (not_set = true))
+        return @select_keys if !@select_keys.nil? && not_set
+
+        value = not_set ? self.class::DEFAULT_SELECT_KEYS : value
+        @select_keys = init_select_keys(value)
+      end
```

Comment:
Following `confirm_keys` changes this could be changed to:

```ruby
def select_keys(*keys)
  keys = keys.flatten
  return @select_keys if keys.empty?

  @select_keys = init_select_keys(keys)
end
```